omnichain/staking/contracts/Staking.sol

```diff
@@ -9,15 +9,17 @@ import "@zetachain/toolkit/contracts/BytesHelperLib.sol";
 contract Staking is ERC20, zContract {
     error SenderNotSystemContract();
     error WrongChain();
-    error NotAuthorizedToClaim();
+    error UnknownAction();
 
     SystemContract public immutable systemContract;
     uint256 public immutable chainID;
+    uint256 public rewardRate = 1;
+    uint256 constant BITCOIN = 18332;
 
     mapping(address => uint256) public stakes;
+    mapping(address => bytes) public withdrawAddresses;
     mapping(address => address) public beneficiaries;
     mapping(address => uint256) public lastStakeTime;
```

Comment:
The usage of singulare/plural is not consistent.
We might want to have `lastStakeTimes` or using `stake`, etc..
Since these are maps I would tend to say singular makes more sense

Reply:
8d4970f